web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -0,0 +1,82 @@
+from impact.v1.helpers.model_helper import ModelHelper
+
+VALID_GENDERS = ["f", "m", "o", "p"]
+
+GENDER_TRANSLATIONS = {
+    "female": "f",
+    "male": "m",
+    "other": "o",
+    "prefer not to state": "p",
+}
+
+INVALID_GENDER_ERROR = ("Invalid gender: '{}'. Valid values are "
+                        "'f' or 'female', 'm' or 'male', "
+                        "'o' or 'other', and 'p' or 'prefer not to state'")
+
+
+class ProfileHelper(ModelHelper):
+    REQUIRED_KEYS = [
+        "gender",
+        ]
+    OPTIONAL_STRING_KEYS = [
+        "bio",
+        "company",
+        "facebook_url",
+        "linked_in_url",
+        "office_hours_topics",
+        "personal_website_url",
+        "phone",
+        "referred_by",
+        "speaker_topics",
+        "title",
+        "twitter_handle",
+        ]
+    OPTIONAL_BOOLEAN_KEYS = [
+        "judge_interest",
+        "mentor_interest",
+        "office_hours_interest",
```

Comment:
could we gather these by inspecting the fields themselves?

Reply:
See comment at the end.